Formula/joern.rb

```diff
@@ -0,0 +1,61 @@
+class Joern < Formula
+  desc "Open-source code analysis platform based on code property graphs"
+  homepage "https://joern.io/"
+  url "https://github.com/joernio/joern/archive/refs/tags/v1.1.1395.tar.gz"
+  sha256 "ffc20cb3c190520bfa8782a79c956375290a022da494314dcef0d58c683203f6"
+  license "Apache-2.0"
+
+  depends_on "php" => :build
+  depends_on "sbt" => :build
+  depends_on "coreutils"
+  depends_on "openjdk"
+
+  def install
+    system "sbt", "stage"
+
+    libexec.install "joern-cli/target/universal/stage/.installation_root"
+    libexec.install Dir["joern-cli/target/universal/stage/*"]
+
+    # Remove incompatible binaries.
+    astgen = "astgen-" + (OS.mac? ? "macos" : "linux")
+    astgen += "-arm" if Hardware::CPU.arm? && OS.mac?
+    (libexec/"frontends/jssrc2cpg/bin/astgen").each_child do |f|
+      f.delete if f.file? && f.basename.to_s != astgen.to_s
+    end
```

Comment:
Does it install prebuilt binaries?

Reply:
@SMillerDev Yes. Does this not meet the acceptable formula policy for the following reason?

> Our policy is that formulae in the core tap homebrew/core must be open-source with a Debian Free Software Guidelines license and either built from source or producing cross-platform binaries (e.g. Java, Mono). Binary-only formulae should go in homebrew/cask.

If it does not, then would you be amenable to the addition of `joernio/astgen` as a resource that I build from source until it meets the notability requirements that would allow me to separate it out as its own formula?

Reply:
Yes, building the resource sounds like a better option. Either that or ignoring the notability requirement. We definitely don't want binary downloads.

Reply:
@SMillerDev if we were to ignore the notability requirement, how does that work? Is that something you do once I open the PR for the new `astgen` formula, or is it an instruction I would need to add to the formula?

Reply:
No, maintainers can merge formulae even if the audit fails. So it would be something I do.

Reply:
Okay, I have opened a PR for the new `astgen` formula (https://github.com/Homebrew/homebrew-core/pull/121356)